content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -558,3 +558,178 @@ while (resultSet.hasNext()) {
 ```
 {{% /tab %}}
 {{< /tabs >}}
+
+# Vault-Named Query Operators
+
+The following is the list of the operators and right operand types for the vault-named query syntax:
+
```

Comment:
Do these match the subsections below?

Reply:
Oh, ok. I think that could be clearer. Should that bulleted list not include ALL of the operators, including those explained further below (perhaps with links)? Also, is "and right operand types" correct?